docs/examples/contours/canny_edge_demo.jl

```diff
@@ -0,0 +1,91 @@
+# ---
+# cover: assets/corner.png
+# title: Canny Edge Detection
+# author: Ashwani Rathee
+# date: 2021-2-1
+# ---
+
+
+# Canny filter is a multi-stage edge detector. It uses a filter based on the 
+# derivative of a Gaussian in order to compute the intensity of the gradients.
+# The Gaussian reduces the effect of noise present in the image. Then, potential
+# edges are thinned down to 1-pixel curves by removing non-maximum pixels of 
+# the gradient magnitude. Finally, edge pixels are kept or removed using hysteresis
+# thresholding on the gradient magnitude.
+
+# Canny has three adjustable parameters: the width of the Gaussian (the noisier
+# the image, the greater the width), and the low and high threshold for the 
+# hysteresis thresholding.
+
+# Importing Packages
+
+using ImageTransformations
+using ImageCore
+using TestImages
+using CoordinateTransformations
+using Rotations
+using ImageFiltering
+using ImageEdgeDetection
+using Noise
+
+# For most purposes, any AbstractArray can be treated as an image.
+# For example, numeric array can be interpreted as a grayscale image.
+
+# First,We create our test image.
+
+img_gray = zeros( Gray{Float64}, 200, 200)
+img_gray[ 50:150, 50:150] .= 1
+mosaicview( img_gray; nrow=1)
```

Comment:
```suggestion
img_gray = zeros(Gray{Float64}, 200, 200)
img_gray[50:150, 50:150] .= 1
mosaicview(img_gray; nrow=1)
```

Reply:
```diff
- mosaicview( img_gray; nrow=1)
+ img_gray #hide #md
```
See tips in: https://johnnychen94.github.io/DemoCards.jl/dev/quickstart/usage_example/julia_demos/2.cover_on_the_fly/#Generate-your-cover-on-the-fly